Source/WebKit/NetworkProcess/NetworkProcessCreationParameters.h

```diff
@@ -82,6 +82,8 @@ struct NetworkProcessCreationParameters {
     Vector<std::pair<WebCore::ProcessIdentifier, WebCore::RegistrableDomain>> allowedFirstPartiesForCookies;
     HashSet<String> localhostAliasesForTesting;
     Vector<WebCore::OrganizationStorageAccessPromptQuirk> storageAccessPromptQuirksData;
+
+    double defaultRequestTimeoutInterval { INT_MAX };
```

Comment:
We should not use special values like this in modern code. use std::optional or WTF::Markable. WTF::Markable is our modern way of doing special values. std::optional is good too but will take more space since it does not rely on a special value.

Reply:
We should consistently use Markable I think but you can feel free to do that in a follow up IF this patch will get cherry-picked to a SU branch (didn't check).

Reply:
Got it. Just updated the patch to use Markable for the both the Network Process and Web Process.